### PR TITLE
Using switch case

### PR DESCRIPTION
**Pull request by subpathdev**
**What type of PR is this?**
/kind cleanup
I think this is a wrong type of pr but I don't know a better one.

**What this PR does / why we need it**:
In my opinion this pr increase the readability of this code part.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by subpathdev** [comment]
Sry I have rebased the source and removed the merge commits. If you want to have merge commits I will fix it.

**Comment by m1093782566** [comment]
@subpathdev 

Please rebase to the HEAD.

**Comment by m1093782566** [comment]
Nice catch! Thanks for cleanup.

File: edge/pkg/devicetwin/dtmodule/dtmodule.go [hunk 1]
```diff
@@ -17,28 +15,30 @@ type DTModule struct {
...
-	} else if strings.Compare(dm.Name, "TwinModule") == 0 {
+	case "TwinModule":
 		dm.Worker = dtmanager.TwinWorker{
 			Group: "TwinModule",
 			Worker: dtmanager.Worker{ReceiverChan: recv,
 				ConfirmChan:   confirm,
 				HeartBeatChan: heartBeat,
 				DTContexts:    dtContext}}
-	} else if strings.Compare(dm.Name, "DeviceModule") == 0 {
+
```
Comment (sids-b) [inline edge/pkg/devicetwin/dtmodule/dtmodule.go]: please remove extra blank line

**Pull request merged**